app/bundles/LeadBundle/Controller/LeadController.php

```diff
@@ -1394,10 +1394,48 @@ public function emailAction($objectId = 0)
                             empty($email['fromname']) ? null : $email['fromname']
                         );
 
+                        // Attachments
+                        $assets = [];
+                        $assetsIds  = (!empty($email['assetAttachments'])) ? $email['assetAttachments'] : [];
```

Comment:
Shorter syntax can be used:
```suggestion
                        $assetsIds  = $email['assetAttachments'] ?? [];
```